subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -1091,6 +1184,10 @@ static int wifi_get_version(uint32_t mgmt_request, struct net_if *iface,
 		return -ENOTSUP;
 	}
 
+	if (!net_if_is_admin_up(iface)) {
+		return -ENETDOWN;
+	}
```

Comment:
I think we could allow the version command to print information even if interface is down.